test/unit/models/classes/search/index/IndexDocumentBuilderTest.php

```
@@ -63,25 +57,20 @@ class IndexDocumentBuilderTest extends TestCase
-    /** @var MockObject|string */
-    private $ontologyMock;
-
-    /** @var core_kernel_classes_Resource|MockObject */
-    private $resourceMock;
-
-    /** @var SearchTokenGenerator|MockObject */
-    private $tokenGeneratorMock;
-
-    /** @var PermissionInterface|MockObject */
-    private $permissionProvider;
+    private ServiceManager|MockObject $serviceLocatorMock;
```

PHPMD (2.15.0)

Avoid unused private fields such as '$serviceLocatorMock'. (undefined)

(UnusedPrivateField)